pkg/resources/cloudconfig/cloudconfig.go

```diff
@@ -43,6 +43,7 @@ func CloudConfig(
 	cloud := cluster.Spec.Cloud
 	switch {
 	case cloud.AWS != nil:
+		DualstackIPFamilies := []string{"ipv6", "ipv4"}
```

Comment:
should be in reversed order - `"ipv4", "ipv6"` - we do not sup[port IPv6 as the primary IP family in KKP yet